fabulaws/ubuntu/instances/base.py

```diff
@@ -160,6 +160,17 @@ def _destroy_volume(self, vol):
 
     @property
     @uses_fabric
+    def server_memory(self):
+        """Returns total server memory, in MB"""
+        if not hasattr(self, '_server_memory'):
+            mem = run('cat /proc/meminfo|grep MemTotal')
+            while '  ' in mem:
+                mem = mem.replace('  ', ' ')
+            self._server_memory = int(mem.split(' ')[1])/1024
```

Comment:
It would be nice to add spaces around the division operator, unless your stylistic conscience has a reason to believe otherwise.